packages/big-design/src/components/Worksheet/editors/CheckboxEditor/CheckboxEditor.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { typedMemo } from '../../../../utils';
+import { Checkbox } from '../../../Checkbox';
+import { Cell, WorksheetItem } from '../../types';
+
+import { CheckboxWrapper } from './styled';
+
+export interface CheckboxEditorProps<Item> {
+  cell: Cell<Item>;
+  onChange(value: unknown): void;
+}
+
+const InternalCheckboxEditor = <T extends WorksheetItem>({ cell, onChange }: CheckboxEditorProps<T>) => {
+  const handleChange = () => {
+    onChange(!cell.value);
+  };
+
+  const checked = cell.value;
```

Comment:
Do we need this const declaration?

Reply:
I was thinking about removing it, so might as well.